### ISSUE TITLE
Key icon does not appear when trying to connect

### ISSUE THREAD

**Issue by realstokezz** [Anton2319/WireGuardExamples#3]
When I click a button, I get the prompt asking if I want to allow vpn connnection. I click allow. And then the vpn is suppose to connect. But I do not see a key icon appear on the status bar. I even checked whatismyip.com to make sure that it works and nope, the ip did not change.

**Comment by Anton2319** [comment]
Can you please provide me with the configuration and code you use? If it is private, I could help you in DMs and post a comment here

**Comment by realstokezz** [comment]
Yes, I would like to send you a copy, shortened version of my code, so that you can read the wire guard configuration and code better without the extras in the way. And yes my source code I would like private, and of course we can comment here too.

**Comment by Anton2319** [comment]
Contact info is available on my website: [https://anton2319.ru](https://anton2319.ru)

**Comment by realstokezz** [comment]
I sent you a message on instagram
